docker/parity_wrapper.sh

```diff
@@ -203,8 +204,12 @@ function runParity() {
 }
 
 function copySpecFileToSharedVolume() {
-  echo "Copying trustline spec file to shared volume"
-  cp /config/trustlines-spec.json /shared/trustlines-spec.json
+  if [[ -d "$SHARED_VOLUME_PATH" ]]; then
+    echo "Copying trustline spec file to shared volume"
+    cp /config/trustlines-spec.json "$SHARED_VOLUME_PATH/trustlines-spec.json"
```

Comment:
there is a slash already in `/shared/` is that not an issue?

Reply:
okay, looks like this should not be an issue apparently.

Reply:
According to this [Stackoverflow answer](https://stackoverflow.com/a/24026057), multiple slashes are treated as a single one, so this should work. I also tested it and it seems to work.